### PR TITLE
feat: ITV-27 remove rooms and all references to it

### PR DESCRIPTION
**Pull request by LBHABEBIC**
Removed the rooms page and all references to it.

Note: There are a few TODOs on this page as the deletion of this page changes the order of the pages but the new pages which replace "rooms" do not exist yet. 

https://hackney.atlassian.net/jira/software/projects/ITV/boards/40/roadmap?selectedIssue=ITV-27

### CODE REVIEW

File: pages/itv/[processRef]/sections.tsx [hunk 1]
```diff
@@ -216,7 +216,7 @@ const usePropertyInspectionStatus = (
       }
 
       const property = propertySet[processRef];
-      const completedFirstStep = property?.rooms !== undefined;
+      const completedFirstStep = property?.laminatedFlooring !== undefined; //TODO change to correct page
```
Comment (mrmartineau) [inline pages/itv/[processRef]/sections.tsx:219]: What should the correct page be? A new one?

Comment (mrmartineau) [reply pages/itv/[processRef]/sections.tsx:219]: Apologies, I just re-read your PR description so I know why the note is there 😄

**Review by mrmartineau — APPROVED** [review]
Looks good, gonna need to fix the conflict before merging though

**Review by LBH-wgreeff — APPROVED** [review]
looks good

**Pull request merged**